repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkCommandLine.scala

```diff
@@ -23,10 +23,15 @@ import scala.Predef._
 /**
  * Command class enabling Spark-specific command line options (provided by
  * <i>org.apache.spark.repl.SparkRunnerSettings</i>).
+ *
+ * @example new SparkCommandLine(Nil).settings
+ *
+ * @param args The list of command line arguments
+ * @param settings The underlying settings to associate with this set of
+ *                 command-line options
  */
 class SparkCommandLine(args: List[String], override val settings: Settings)
```

Comment:
Usually out of abundance of caution we mark it on the class level as well, even if everything bytecode-exposed inside of the class is also marked. We tend to err on the side of over communication with this.